src/services/OfficeRnDService.ts

```diff
@@ -99,6 +102,16 @@ export class OfficeRnDService {
     } while (currNextPointer != '');
     return teamsArray;
   };
+
+  private getMembers = async (bookings: OfficeRndBooking[]) => {
+    let memberPromises = new Array<Promise<OfficeRnDMember>>();
+    bookings.forEach(async (booking) => {
+      memberPromises.push(this.fetchWithToken<OfficeRnDMember>(
+        `${this.BASE_API_URL}/members/${booking.member}`
+      ));
+    });
```

Comment:
It's better to use a map there: 
```typescript
const memberPromises = bookings.map(booking => this.fetchWithToken<OfficeRnDMember>(
        `${this.BASE_API_URL}/members/${booking.member}`
      ))
 ```

And it might be good to have a function 
```typescript
private getMember = (memberId: string) => this.fetchWithToken<OfficeRnDMember>(
        `${this.BASE_API_URL}/members/${memberId}`
      )
```
      
      
And reuse this function in the map
```
const memberPromises = bookings.map(booking => this.getMember(booking.member))

Reply:
Fair enough. That does seem easier to read